### PR TITLE
fix: hide double logo

### PR DESCRIPTION
**Pull request by felix-ico**
this should fix https://github.com/telekom/scale/issues/1448

### CODE REVIEW

File: packages/components/src/components/telekom/logo/logo.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export class Logo {
   /** (optional) Hide all logo related titles */
   @Prop() logoHideTitle?: boolean;
   @Prop() logoAriaDescribedBy?: string;
+  @Prop() logoAriaHide?: boolean = false;
```
Comment (acstll) [inline packages/components/src/components/telekom/logo/logo.tsx:55]: Wouldn't you call this one `logoAriaHidden` following the other similar "inner aria" props?

**Comment by felix-ico** [comment]
@acstll i pushed a commit to improve the naming, is the rest of the code fine with you to merge?

**Review by nowseemee — APPROVED** [review]
👍🏻

**Comment by acstll** [comment]
@felix-ico also 👍 — sorry for the late reply!

**Pull request merged**